### ISSUE TITLE
Can't mop up "Clean water (dirty)" on the floor

### ISSUE THREAD

**Issue by thirite2** [CleverRaven/Cataclysm-DDA#28762]
**Describe the bug**  
In the middle of the night, my 3L Glass Jar of Clean water broke somehow, when my character was sleeping. The Clean water spilled on the floor and became "Clean water (dirty)" and when I try to mop it up, the game does not register that there is anything there to be mopped up. When I try to reproduce the bug by spilling clean water on the floor, or smashing a glass jar with clean water in it, it does not reproduce the bug.

**To Reproduce**  
- I have no idea. :( Savegame is attached below, with bugged water on the floor in the evac shelter I've turned into a base.

**Expected behavior**  
The water should be able to be mopped up like any other spilt liquid.

**Screenshots**  
https://i.imgur.com/yPHiAfb.png

**Versions and configuration:**
 - OS: Win7 64bit
 - Game Version: 0.D-572-gc62f18a (tiles)
 - Mods loaded: Disable NPC Needs, Simplified Nutrition, Extended Realistic Guns, Generic Guns, Growable Pots, Makeshift Items Mod, Medieval and Historic Content, More Survival Tools, Modular Turrets, Salvaged Robots, Alternative Map Key, More Locations, Fuji's More Buildings, Urban Development, Boats, Folding Parts Pack, Vehicle Additions Pack, Tanks and Other Vehicles, No Fictional Guns, No Flaming Weapons, No Sci-Fi Equipment, Classes and Scenarios Mod

**Additional context**  
Save file with the unmoppable puddle of water in the evac shelter
https://cdn.discordapp.com/attachments/365907426270117888/556569801011757056/Eagledale.zip

**Comment by Night-Pryanik** [comment]
No surprise you can't mop it up, because it's the Schrödinger Water, being clean and dirty at the same time!

**Comment by KongMD-Steam** [comment]
Anecdotally, I have encountered this as well several times in my game on Build 8627. For example, in a mansion I found, there was a furniture item (dresser, maybe?) that spawned with a container of Riesling beer. I never saw the actual container, but discovered the beer (dirty) on the floor as I was exploring the mansion.

**Comment by umsretuza** [comment]
I made [a post about this on the forums about a week ago](https://discourse.cataclysmdda.org/t/is-there-any-way-to-remove-liquid-off-the-ground/19195/5?u=retuza). Strange behavior that I never really figured out. After reading this post it does bring me back to my original post where I wrote:

>  after they somehow lost their original containers. (in winter)

now it feels like there's definitely some monkey business going on.
